lib/rust/flexer/src/group.rs

```diff
@@ -21,14 +21,18 @@ pub mod rule;
 #[derive(Clone,Debug,Default)]
 pub struct Registry {
     /// The groups defined for the lexer.
-    groups: Vec<Group>
+    groups:Vec<Group>
```

Comment:
Btw, in our code, when defining structs, we use space on BOTH SIDES of `:` . It is specified in the code style (as far as I know), but I know its not "unified" with other rules. Maybe we should change it to follow your style? Dont know. Anyway, I believe that here, keeping spaces around `:` is more clear for the reader. Still, this is so minor thing, that you can skip this comment.